media/js/firefox/sticky-promo.js

```diff
@@ -14,6 +14,14 @@
             return;
         }
 
+        // If the user is on Firefox and is NOT supposed to see sticky promo.
```

Comment:
This logic was written in such a way as to not require any modifications to the other pages who are already using the sticky promo. This means **the default behavior is to be visible to Firefox users** (and show them a list of other Firefox products).

Reply:
👍 Nice backwards compatability.